docs/man/git-lfs-fetch.1.ronn

```diff
@@ -126,6 +127,11 @@ What changes are considered 'recent' is based on a number of gitconfig options:
 
   `git lfs fetch upstream`
 
+* Fetch all the LFS objects from the default remote that are referenced by any
+  commit in the `master` and `develop` branches
+
+  `git lfs fetch --all master develop`
```

Comment:
I think we'd want this to say `git lfs fetch --all origin master develop`.

Reply:
Also correct, thanks for finding this one as well!